src/app/components/Login/User.tsx

```diff
@@ -1,15 +1,23 @@
+import { Dropdown, MenuProps } from 'antd';
+import { LogoutOutlined, SettingOutlined } from '@ant-design/icons';
 import { signIn, signOut, useSession } from 'next-auth/react';
 import { UserCircleIcon } from '@heroicons/react/24/outline';
-import { ActionUserIcon } from '@/components';
-import styled from 'styled-components';
+import { ActionIcon } from '@/components';
 
-const Image = styled.img`
-  display: flex;
-  align-items: center;
-  text-align: center;
-  max-width: 40px;
-  border-radius: 50%;
-`;
+const items: MenuProps['items'] = [
+  {
+    key: '2',
+    label: 'settings',
+    icon: <SettingOutlined />,
+  },
```

Comment:
this is optional, left it here for the sake of example, let me know if I should remove or replace with smth

Reply:
It can stay, np